action.yml

```diff
@@ -0,0 +1,35 @@
+
+# action.yml
+name: "Lazy Action Nodejs"
+description: "Github action to perform build, test , scan and generate image."
+inputs:
+  dockerfile_dir_path:
+    description: "Directory path to the dockerfile"
+    required: true
+  npm_test_placeholder:
```

Comment:
What is npm test placeholder?

Reply:
This placeholder variable is to pass npm test tag like test or test:coverage

Reply:
Can you call it something like npm test script name?

Reply:
The reason why I'm skeptical of placeholder is because placeholder as the word says is a temporary holder